third_party/3/jwt/__init__.pyi

```diff
@@ -0,0 +1,50 @@
+from typing import Mapping, Any, Optional, Union
+
+from . import algorithms
+
+def decode(jwt: Union[str, bytes], key: Union[str, bytes] = ...,
+           verify: bool = ..., algorithms: Optional[Any] = ...,
+           options: Optional[Mapping[Any, Any]] = ...,
+           **kwargs: Any) -> Mapping[str, Any]: ...
+
+def encode(payload: Mapping[str, Any], key: Union[str, bytes],
+           algorithm: str = ..., headers: Optional[Mapping[str, Any]] = ...,
+           json_encoder: Optional[Any] = ...) -> bytes: ...
+
+def register_algorithm(alg_id: str,
+                       alg_obj: algorithms.Algorithm) -> None: ...
+
+def unregister_algorithm(alg_id: str) -> None: ...
+
+class InvalidTokenError(Exception): pass
+
```

Comment:
I'd remove the blank lines between the various exception classes too (but keep them between this block and the functions and non-exception definitions).